hera_pspec/pspecdata.py

```diff
@@ -3999,7 +4012,11 @@ def pspec_run(dsets, filename, dsets_std=None, cals=None, cal_flag=True,
     # Construct dataset pairs to operate on
     Ndsets = len(dsets)
     if dset_pairs is None:
-        dset_pairs = list(itertools.combinations(range(Ndsets), 2))
+        # this fails silently if we only provided a single dset.
```

Comment:
this comment is out of place, b/c it is referring to the old behavior, which is now fixed, so it won't match what will be in the codebase once this is merged. perhaps something like "# define dset_pairs if not fed"

Reply:
Will fix this.